dbclient/WorkspaceClient.py

```diff
@@ -261,7 +262,9 @@ def download_notebook_helper(self, notebook_path, export_dir='artifacts/'):
         get_args = {'path': notebook_path, 'format': self.get_file_format()}
         if self.is_verbose():
             print("Downloading: {0}".format(get_args['path']))
-        resp = self.get(WS_EXPORT, get_args)
+        (checkpoint_available, resp) = self.checkpoint_get(WM_EXPORT, WORKSPACE_NOTEBOOK_OBJECT, notebook_path, WS_EXPORT, get_args)
```

Comment:
Does `checkpoint_available` mean the notebook was in the checkpoint file? It's hard to understand what this means from the variable name.

Reply:
renamed it to `found_in_checkpoint`